src/data/vagrant.json

```
@@ -74,5 +74,43 @@
 			}
 		]
 	},
-	"navigationHeaderItems": {}
+	"navigationHeaderItems": {
+		"documentation": [
+			{
+				"icon": "docs",
+				"label": "Introduction",
+				"pathSuffix": "docs"
+			},
+			{
+				"icon": "docs",
+				"label": "General Documentation",
+				"pathSuffix": "docs"
+			},
+			{
+				"icon": "docs",
+				"label": "Vagrant Cloud",
+				"pathSuffix": "cloud"
+			}
```

Since [/vagrant/intro](https://dev-portal-git-onboard-vagrant-hashicorp.vercel.app/vagrant/intro) and [/vagrant/vagrant-cloud](https://dev-portal-git-onboard-vagrant-hashicorp.vercel.app/vagrant/vagrant-cloud) are the current URLs for those sections of docs, let's set those as the `pathSuffix` values here for them.

```suggestion
			{
				"icon": "docs",
				"label": "Introduction",
				"pathSuffix": "intro"
			},
			{
				"icon": "docs",
				"label": "General Documentation",
				"pathSuffix": "docs"
			},
			{
				"icon": "docs",
				"label": "Vagrant Cloud",
				"pathSuffix": "vagrant-cloud"
			}
```